.wordlist.txt

```diff
@@ -121,6 +122,7 @@ frontend
 fullscreen
 FuguHub
 Gbit
+german
```

Comment:
I think "German language" (always capitalised) is correct, isn't it?

Reply:
I am not sure but change it.